webui/react/src/shared/components/Avatar/Avatar.test.tsx

```diff
@@ -1,20 +1,20 @@
-import { render, screen, waitFor } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { TooltipProps } from 'antd/es/tooltip';
 import React from 'react';
 
 import StoreProvider from 'contexts/Store';
+import { DarkLight } from 'shared/themes';
```

Comment:
I think within shared it might be more portable if we use relative imports. (not that we use the portability today)

Reply:
actually thought about this some more, if Avatar.test.tsx is moved around, the imports will remain valid and functional with the current setup, where as relative paths would require updating...

Reply:
true. the portability I was talking about was if this shared dir was hosted elsewhere, a different dir path or name. but yeah we don't use that today so works either way.

Reply:
ah gotcha